net-irc/hexchat/hexchat-2.12.1.ebuild

```diff
@@ -0,0 +1,166 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_3 python3_4 python3_5 )
+inherit eutils fdo-mime gnome2-utils mono-env multilib python-single-r1
+
+DESCRIPTION="Graphical IRC client based on XChat"
+HOMEPAGE="https://hexchat.github.io/"
+SRC_URI="https://dl.hexchat.net/hexchat/${P}.tar.xz"
+
+LICENSE="GPL-2 plugin-fishlim? ( MIT )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+IUSE="dbus debug +gtk libcanberra libnotify libproxy libressl nls perl +plugins plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
+REQUIRED_USE="plugins? ( python? ( ${PYTHON_REQUIRED_USE} ) )"
+
+DEPEND="dev-libs/glib:2
+	dbus? ( >=dev-libs/dbus-glib-0.98 )
+	gtk? ( x11-libs/gtk+:2 )
+	libcanberra? ( media-libs/libcanberra )
+	libproxy? ( net-libs/libproxy )
+	libnotify? ( x11-libs/libnotify )
+	nls? ( virtual/libintl )
+	plugins? (
+		perl? ( >=dev-lang/perl-5.8.0 )
+		plugin-sysinfo? ( sys-apps/pciutils )
+		python? ( ${PYTHON_DEPS} )
+	)
+	spell? ( app-text/iso-codes )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl )
```

Comment:
both openssl and libressl need `:0=`
https://wiki.gentoo.org/wiki/Project:LibreSSL